src/main/java/com/hystrix/configurator/core/HystrixConfigurationFactory.java

```diff
@@ -105,6 +119,22 @@ private void registerDefaultProperties(HystrixDefaultConfig defaultConfig) {
         configureProperty("hystrix.command.default.metrics.healthSnapshot.intervalInMilliseconds", defaultConfig.getMetrics().getHealthCheckInterval());
     }
 
+    private void registerCommandProperties(String group, String command) {
+        val threadPool = config.getPools().containsKey(sanitizeString(group))
+                ? toCommandThreadPool(group)
+                : defaultConfig.getThreadPool();
+
+        HystrixCommandConfig commandConfig = HystrixCommandConfig.builder()
+                .name(command)
+                .semaphoreIsolation(false)
```

Comment:
So this cannot be used to configure semaphone pools?

Reply:
by default we only create Thread Isolation. This behaviour has not been changed.